modules/models/src/main/java/org/eclipse/xpanse/modules/models/policy/servicepolicy/ServicePolicy.java

```diff
@@ -45,6 +45,9 @@ public class ServicePolicy {
     @Schema(description = "The id of registered service template which the policy belongs to.")
     private UUID serviceTemplateId;
 
+    @Schema(description = "The flavor name which the policy belongs to.")
```

Comment:
please extend the description here to -

The flavor name which the policy belongs to. If a flavor is not provided, then the policy will be executed for during service deployment of all flavors.

Reply:
Updated.